cabot/settings.py

```diff
@@ -329,6 +329,6 @@
 TEST_RUNNER = 'xmlrunner.extra.djangotestrunner.XMLTestRunner'
 TEST_OUTPUT_DIR = os.environ.get('TEST_OUTPUT_DIR', '.')
 
-DISABLE_LOGIN = os.environ.get('DISABLE_LOGIN', False)
+DISABLE_LOGIN = os.environ.get('DISABLE_LOGIN', 'False').lower() in ['true', 'yes', '1']
```

Comment:
Why not just True/False? Seems a little confusing to allow different types of values?

Reply:
Just trying to catch common truthy values.